scaffolder-templates/delete-tenant/template.yaml

```diff
@@ -68,6 +73,7 @@ spec:
 
     - id: logAction
       name: Logging action
+      if: ${{ parameters.s3_emptied and parameters.alarm_removed }}
```

Comment:
Im not 100% confident on this change.

Reply:
Can you change the name and id of this action also, it's confusing now